From: Wei Tang <tangwei@cmss.chinamobile.com>
Date: Tue, 24 Nov 2015 09:58:46 +0800
Subject: block: do not initialise statics to 0 or NULL
Git-commit: 1fe8f348416b3fb35ea3f24fa92bb1d29ffe7b0b
Patch-mainline: v4.5-rc1
References: FATE#319965, bsc#964944

This patch fixes the checkpatch.pl error to genhd.c:

ERROR: do not initialise statics to 0 or NULL

Signed-off-by: Wei Tang <tangwei@cmss.chinamobile.com>
Signed-off-by: Jens Axboe <axboe@fb.com>
Acked-by: Johannes Thumshirn <jthumshirn@suse.com>
---
 block/genhd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/block/genhd.c b/block/genhd.c
index e5cafa5..78140b4 100644
--- a/block/genhd.c
+++ b/block/genhd.c
@@ -1421,7 +1421,7 @@ static DEFINE_MUTEX(disk_events_mutex);
 static LIST_HEAD(disk_events);
 
 /* disable in-kernel polling by default */
-static unsigned long disk_events_dfl_poll_msecs	= 0;
+static unsigned long disk_events_dfl_poll_msecs;
 
 static unsigned long disk_events_poll_jiffies(struct gendisk *disk)
 {

